packages/jest-runtime/src/__tests__/runtime_compile_module.test.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+'use strict';
```

Comment:
no need when using `import/export` - it's implicitly strict

Reply:
Good catch - this was just copied from another test file.

Reply:
a670812d922080e6d4c70bfb292456b705ebac3d